src/um/oaidl.rs

```diff
@@ -689,7 +689,39 @@ interface IRecordInfo(IRecordInfoVtbl): IUnknown(IUnknownVtbl){
         pvRecord: PVOID,
     ) -> HRESULT,
 }}
-pub enum ITypeComp {} // FIXME
+pub type LPTYPECOMP = *mut ITypeComp;
+ENUM!{enum DESCKIND {
+    DESCKIND_NONE = 0, 
+    DESCKIND_FUNCDESC = DESCKIND_NONE + 1, 
+    DESCKIND_VARDESC = DESCKIND_FUNCDESC + 1, 
+    DESCKIND_TYPECOMP = DESCKIND_VARDESC + 1, 
+    DESCKIND_IMPLICITAPPOBJ = DESCKIND_TYPECOMP + 1, 
+    DESCKIND_MAX = DESCKIND_IMPLICITAPPOBJ + 1,
+}}
+UNION!{union BINDPTR {
+    [usize; 1],
+    Lpfuncdesc Lpfuncdesc_mut: LPFUNCDESC, 
+    Lpvardesc Lpvardesc_mut: LPVARDESC, 
+    Lptcomp Lptcomp_mut: LPTYPECOMP,
```

Comment:
In the headers these pointers types are all `*mut BLAH` not `LPBLAH`, and the variants themselves are lowercase.